### ISSUE TITLE
Missing/non-functional drawables

### ISSUE THREAD

**Issue by rjwut** [DeviateFish/ingress-model-viewer#36]
The following objects appear not to work; either no Drawables are defined for them, or nothing renders when you attempt to draw them:

- Beacon
- Circle K Power Cube
- Fracker
- Key
- Key Locker
- Lawson Power Cube
- Media

Additional needs:
- A way to specify beacon type
- XM glow color constant for chaotic matter-based items (frackers, key lockers)
- Color constants for the five key locker colors

**Comment by DeviateFish** [comment]
Feel free to put up a PR for the drawables definitions (except beacon, that one might need special handling).  

See [`manifest/assets.json`](https://github.com/DeviateFish/ingress-model-viewer/blob/master/manifest/assets.json) and [`src/constants.js`](https://github.com/DeviateFish/ingress-model-viewer/blob/master/src/constants.js) for how to define those.  Should be easy enough to follow the pattern for all of the inventory/world items.

Beacons will probably require special handling, but I haven't exactly kept up with the latest anti-obfuscation of the ingress client, so it could be a pretty big task to reverse engineer them and figure out how to implement them.

The color constants will face a similar issue, as well.  I'd recommend contacting someone who does regular teardowns to locate those for you... *cough*

**Comment by DeviateFish** [comment]
Here is a recent example of adding a couple new items: 

[`manitfest/assets.json`](https://github.com/DeviateFish/ingress-model-viewer/pull/34/files#diff-7b52186349bd22e6a66f84d9dcd48b3c) and [`src/constants.js`](https://github.com/DeviateFish/ingress-model-viewer/pull/34/files#diff-a7748843fbda5167d065231e3a2e7a1f)

**Comment by rjwut** [comment]
So I've gotten Circle K/Lawson power cubes, frackers and key lockers working. I've also added in constants for the key locker colors and the chaotic XM glow color, although the latter is estimated and not actually read from Ingress. I ran into problems trying to get keys and media to work, and as you suggested I haven't attempted beacons. I'll be making a pull request soon.

**Comment by rjwut** [comment]
Pull request created: #37

**Comment by DeviateFish** [comment]
@rjwut Did you ever get the canonical dark XM color/key capsule colors from the apk?

**Comment by rjwut** [comment]
Not yet. I'll poke the teardown guy again.

**Comment by DeviateFish** [comment]
👍

**Comment by rjwut** [comment]
Actually, the key capsule colors are correct. It's just the chaotic matter color that is an estimate.

**Comment by rjwut** [comment]
According to the teardown guy:
```
coreGlowChaotic: vec4.fromValues(1, 0.35, 0, 1)
coreGlowChaoticAlt: vec4.fromValues(0.66, 0.1, 0.09, 1)
```

**Comment by DeviateFish** [comment]
Closed by #47
